thoth/slo_reporter/sli_apis/sli_user_api.py

```diff
@@ -23,11 +23,16 @@
 
 import numpy as np
 
+import pandas as pd
+
 from typing import Dict, List, Any
 
 from thoth.slo_reporter.sli_base import SLIBase
 from thoth.slo_reporter.sli_template import HTMLTemplates
 from thoth.slo_reporter.configuration import Configuration
+from thoth.slo_reporter.utils import retrieve_thoth_sli_from_ceph
+
+from io import StringIO
```

Comment:
Instead of importing this in all classes, why don't we import it once in `utils.py` and we move the similar logic there?

so moving from:
```
        retrieved_data = retrieve_thoth_sli_from_ceph(self.configuration.ceph_sli, sli_path)
        data = StringIO(retrieved_data)
        last_week_data = pd.read_csv(data, names=self.total_columns)
```

to 
```
        last_week_data = retrieve_thoth_sli_from_ceph(self.configuration.ceph_sli, sli_path, names=self.total_columns)
```

therefore including same logic in `retrieve_thoth_sli_from_ceph`, WDYT?

Reply:
Thanks a lot @pacospace , PTAL https://github.com/thoth-station/slo-reporter/pull/130/commits/2ec44898c0b3594f912663357235840a5c359cb0